services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/PathHelper.groovy

```diff
@@ -68,8 +67,11 @@ class PathHelper {
             throw new Exception("Unable to make some path more preferable because both paths use same ISLs")
         }
         log.debug "ISL to avoid: $islToAvoid"
-        northbound.updateLinkProps([islUtils.toLinkProps(islToAvoid, ["cost": NOT_PREFERABLE_COST]),
-                                    islUtils.toLinkProps(islToAvoid.reversed, ["cost": NOT_PREFERABLE_COST])])
+        def maxAvailableCost = morePreferableIsls.findAll { it != islToAvoid }.collect {
```

Comment:
morePreferableIsls cannot contain islToAvoid, findAll is useless

Reply:
yes, my fault